tracks.yaml

```diff
@@ -1,4 +1,28 @@
 tracks:
+  kinetic:
+    actions:
+    - bloom-export-upstream :{vcs_local_uri} :{vcs_type} --tag :{release_tag} --display-uri
+      :{vcs_uri} --name :{name} --output-dir :{archive_dir_path}
+    - git-bloom-import-upstream :{archive_path} :{patches} --release-version :{version}
+      --replace
+    - git-bloom-generate -y rosrelease :{ros_distro} --source upstream -i :{release_inc}
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name ubuntu
+    # - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+    #   -i :{release_inc} --os-name debian --os-not-required
+    - git-bloom-generate -y rosrpm --prefix release/:{ros_distro} :{ros_distro} -i
+      :{release_inc}
+    devel_branch: null
+    last_version: 0.2.0
+    name: upstream
+    patches: kinetic_maintainer_update
```

Comment:
once the renaming of the patch folders has been made, this should be updated to match the folder name (same in the lunar track)